src/Common/Acl/AclExtended.php

```diff
@@ -236,9 +237,10 @@ public static function setUserAro($arr_group_titles, $user_name, $first_name, $m
         //see if this user is gacl protected (ie. do not allow
         //removal from the Administrators group)
         require_once(dirname(__FILE__) . '/../../../library/user.inc');
-        require_once(dirname(__FILE__) . '/../../../library/calendar.inc');
-        $userNametoID = getIDfromUser($user_name);
-        if (checkUserSetting("gacl_protect", "1", $userNametoID) || $user_name == "admin") {
+
+        $userNameToID = (new UserService())->getIdByUsername($user_name);
```

Comment:
also need to adjust the 2 places above in this PR that use the getIdByUsername() now that returning the user id directly
(then this is ready to come into codebase :) )